src/codegen.cpp

```diff
@@ -427,7 +427,10 @@ void jl_dump_objfile(char *fname, int jit_model)
     std::string err;
     raw_fd_ostream OS(fname, err);
 #endif
+
+#ifndef LLVM37 // 3.7 simplified formatted output
     formatted_raw_ostream FOS(OS);
+#endif
```

Comment:
Agreed though didn't spend the time to see what the ramifications of that
assignment might be. In c++ such statements can hide a world of complexity.
I'll take a gander today.

On Thursday, April 16, 2015, Jameson Nash <notifications@github.com> wrote:

> In src/codegen.cpp
> <https://github.com/JuliaLang/julia/pull/10845#discussion_r28483553>:
>
> >      formatted_raw_ostream FOS(OS);
> > +#endif
>
> stylistically, I think this would be better with an else clause here,
> rather than copying code below. perhaps something like:
>
> #else
> auto FOS = OS;
> #endif
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaLang/julia/pull/10845/files#r28483553>.
>


-- 
*Sebastian Good*